jwst/msaflagopen/msaflag_open.py

```diff
@@ -43,9 +43,12 @@ def do_correction(input_model, shutter_refname, wcs_refnames):
     """
     # Create a list of failed open slitlets from the msaoper reference file
     failed_slitlets = create_slitlets(shutter_refname)
+    log.info("%d failed open shutters", len(failed_slitlets))
 
     # Flag the stuck open shutters
-    output_model = flag(input_model, failed_slitlets, wcs_refnames)
+    with warnings.catch_warnings():
+        warnings.filterwarnings("ignore", category=RuntimeWarning, message="Invalid interval")
```

Comment:
I could move this to a lower level call but I did not because it was wrapped in a loop.